.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,39 @@
+name: Build wheels and archive
+
+on:
+- pull_request
+- workflow_dispatch
```

Comment:
What's the purpose of this workflow? Check wheel builds for every PR?

Reply:
I plan to remove the `pull request` and just have it available for `workflow_dispatch` for manual testing/debug of builds.  It's overkill on each PR.

Reply:
Maybe we could set up a cron dispatch to check the builds work every month or something?

Reply:
Let's go with `workflow_dispatch` for now

Reply:
Leaving the possibility of cron dispatch for a future PR - needs somewhere to notify about failures